frontend/src/metabase/core/components/TabRow/TabRow.styled.tsx

```diff
@@ -21,3 +23,29 @@ export const TabList = styled(BaseTabList)`
     margin-right: 2rem;
   }
 `;
+
+interface ScrollButtonProps {
+  direction: "left" | "right";
+}
+
+export const ScrollButton = styled.button<ScrollButtonProps>`
```

Comment:
Copied this from the [PR](https://github.com/metabase/metabase/pull/24493) where it was deleted.